### ISSUE TITLE
Feature: filter bar in admin

### ISSUE THREAD

**Issue by cyberscribe** [tombenner/wp-mvc#78]
It would be great to have a filter bar to filter on various fields in the admin. This would make it much more useful for implementations with a lot of records.

**Comment by polcodemichalkrawczyk** [comment]
Yep, really useful feature. I used https://datatables.net/ library to achieve this.
Not sure if this could be somehow included into wp-mvc core.

**Comment by cyberscribe** [comment]
I was thinking more just to implement the standard WordPress filters for now, but datatables is great too.
